lua/legendary/ui/format.lua

```diff
@@ -38,11 +38,15 @@ function M.default_format(item)
       item.description,
     }
   elseif Toolbox.is_function(item) then
+    -- stylua: ignore start
     return {
-      Config.icons.fn,
+      item--[[@as keymap ]]:modeSwitched()
+        and table.concat(item--[[@as Keymap]]:modes(), ', ')
+        or Config.icons.fn,
```

Comment:
```suggestion
      table.concat(item--[[@as Function]]:modes() or { Config.icons.fn }, ', '),
```

Reply:
Please, look carefully what you suggest; this introduces a bug; `item:modeSwitched()` is essential here.

Reply:
It doesn't introduce a bug if you also tweak the implementation of `modes()` as I suggested.